source/includes/cloudstack/_egress_rules.md

```diff
@@ -0,0 +1,154 @@
+### Egress rules
+
+Manage outbound traffic rules for isolated networks
+
+#### List egress rules
+```shell
+curl -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/compute-on/test_area/egressrules"
+
+# Response example:
+```
+```json
+{
+    "data": [
+       {
+            "id": "f6c7f86f-1e28-42fd-804f-0416a201a778",
+            "networkId": "858d3ab6-04c9-43ef-9e83-07f28df59323",
+            "sourceCidr": "10.2.1.0/25",
+            "destinationCidr": "",
+            "protocol": "ALL",
+            "state": "Active",
+            "allPortsAllowed": false
```

Comment:
I wonder if we really need this boolean in the API doc. Are you requiring this flag to be set in order for the ports to be omitted?

Reply:
Yes. If it's true, no ports are set

Reply:
I think we could omit it from the doc. It's not very useful since you have the protocol ALL.

Reply:
ok. omitted.